### ISSUE TITLE
Fight..

### ISSUE THREAD

**Issue by PicassoCT** [Anarchid/blender2lus#11]
Begun the model format war has - in the left corner (Beherith) with s3o, the spring format. In the right corner (Anarchid) (DAE) .

He who has more features and better support for the gamedevs - shall win the future.

Are you ready?

**Comment by PicassoCT** [comment]
https://github.com/Beherith/Skeletor_S3O/issues/1

**Comment by Anarchid** [comment]
The core approach is actually kind of format-agnostic. It creates bones that match the axes of each piece. It will need a minor addition for dae - to read the initial orientations at start - and that's all.

This has been a collaboration, kind of :)

**Comment by PicassoCT** [comment]
But you wanted the s3o to die.. and now it will live..

**Comment by PicassoCT** [comment]
Glorious victory awaits

**Comment by Beherith** [comment]
Yep, Anarchid, feel free to PR anything that will help your DAE/LUA workflow. There is currently a button called 'Correctly rotate S3O'. Feel free to use that as it is unused by Skeletor (I overwrote s3o import instead).

**Comment by PicassoCT** [comment]
I loath these moments of enthusiastic harmony. I had money invested in this dog fights. Now i have to leave the country again and get a new name.
